leaf-ui/js/object/Class.js

```diff
@@ -343,7 +343,36 @@ class AnalysisResult {
         this.colorVis; //color visualization for analysis mode  
         this.selectedTimePoint; //find where slider is initialized and set timepoint in here. Also place it in update function
         this.maxTimePoint;
+
+
+        this.elements;
+        this.numOfElements;
+        this.timeScale;
+        this.relativeTime;
     }
+    // Parse results from backend
+    initFromBackEnd = function (analysisResults) {
+        this.elements = [];
+        this.numOfElements = Number(analysisResults.elementList.length);
+        this.timeScale = Number(analysisResults.timePointPath.length) - 1;
+        this.relativeTime = [];
+
+        for (var i = 0; i < analysisResults.timePointPath.length; i++) {
```

Comment:
it doesn't look like you are doing anything significant to this variable either - would it just be a copy of the already existing `timePointPath` variable? Because in that case, just rework the other code to accept the `timePointPath` variable instead of `relativeTime`